app/src/main/kotlin/com/wire/android/feature/conversation/content/ui/ConversationActivity.kt

```diff
@@ -36,7 +37,7 @@ class ConversationActivity : AppCompatActivity(R.layout.activity_conversation) {
     }
 
     companion object {
-        fun newIntent(context: Context, conversationId: String, conversationTitle: String) =
+        fun newIntent(context: Context, conversationId: ConversationID, conversationTitle: String) =
```

Comment:
Here the `ConversationID` can be mapped to two different bundle extras, to avoid having `android` dependencies on domain modules.